### PR TITLE
Added doc_link and help_text to technologies

### LINKED ISSUE #6: Change migrations for db schema
## `technologies` table
* add doc_link as string
* add help_text as text

## Useage for techHelp dialog
Requirements:
- Show a list of technology we can offer documentation/info
- description of technology (short blurb on what it can do and what docs we are presenting)
- documentation links

### example
id: 1
name: Bot Framework
projects: [Project]
help_text: Bots are a great way to create artificial human interaction with your users. Combine bots with Cognitive Services and you will be able to create an intelligent chatbot capable of understanding language intentions and much more.
doc_link: https://dev.botframework.gettingStarted.example

### PR DESCRIPTION
**Pull request by howlowck**
closes #6

### CODE REVIEW

File: migrations/20180211220023_add_technologies_table_doc_help.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+exports.up = (knex, Promise) =>
+  knex.schema.table('technologies', table => {
+    table.string('doc_link')
+    table.text('help_text')
```
Comment (kevinleung23) [inline migrations/20180211220023_add_technologies_table_doc_help.js:4]: What is the significance of .text vs .string?

File: seeds/020-technologies.js [hunk 2]
```diff
@@ -11,5 +11,5 @@ exports.seed = (knex, Promise) =>
         {name: 'Azure Container Services'},
         {name: 'Azure Iothub'},
         {name: 'CNTK'},
-        {name: 'Bot Framework'}
+        {name: 'Bot Framework', doc_link: 'https://dev.botframework.gettingStarted.example', help_text: ' Bots are a great way to create artificial human interaction with your users. Combine bots with Cognitive Services and you will be able to create an intelligent chatbot capable of understanding language intentions and much more.'}
```
Comment (kevinleung23) [inline seeds/020-technologies.js]: Unnecessary space in help_text string. (first character)

Comment (kevinleung23) [reply seeds/020-technologies.js]: Even though this is much data for now..

**Comment by kevinleung23** [comment]
Let a few comments.

File: migrations/20180211220023_add_technologies_table_doc_help.js (hunk 1 above)
Comment (howlowck) [reply migrations/20180211220023_add_technologies_table_doc_help.js:4]: `string` is a varchar up to 255 characters (i think).  `text` can be really long text, limited by the DBMS.

File: seeds/020-technologies.js (hunk 2 above)
Comment (howlowck) [reply seeds/020-technologies.js]: good catch! deleted the space

**Review by kevinleung23 — APPROVED** [review]
:shipit:

**Pull request merged**